examples/persistence_mongodb.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to create a custom checkpointer using MongoDB\n",
+    "\n",
+    "When creating LangGraph agents, you can also set them up so that they persist their state. This allows you to do things like interact with an agent multiple times and have it remember previous interactions.\n",
+    "\n",
+    "This example shows how to use `MongoDB` as the backend for persisting checkpoint state.\n",
+    "\n",
+    "NOTE: this is just an example implementation. You can implement your own checkpointer using a different database or modify this one as long as it conforms to the `BaseCheckpointSaver` interface."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Checkpointer implementation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture --no-stderr\n",
+    "%pip install -U psycopg psycopg-pool langgraph pymongo"
```

Comment:
please delete these `psycopg psycopg-pool` as they're not used in the notebook

Reply:
could you also suppress the error outputs from installation in the final notebook? so the errors don't show up in the documentation